rewrite-yaml/src/main/java/org/openrewrite/yaml/format/IndentsVisitor.java

```diff
@@ -95,12 +99,24 @@ public IndentsVisitor(IndentsStyle style, @Nullable Tree stopAfter) {
                 y = y.withPrefix(indentComments(y.getPrefix(), indent));
             }
         }
+
+        if (y instanceof Yaml.Scalar && y.getMarkers().findFirst(MultilineScalarChanged.class).isPresent()) {
+            int indentValue = indent;
+
+            if (!y.getMarkers().findFirst(MultilineScalarChanged.class).get().isAdded() && indent != 0) {
+                indentValue = indent + style.getIndentSize();
+            }
+
+            String newValue = ((Yaml.Scalar) y).getValue().replaceAll("\\R", "\n" + StringUtils.repeat(" ", indentValue));
+            y = ((Yaml.Scalar) y).withValue(newValue);
+        }
+
         return y;
     }
 
     private boolean isUnindentedTopLevel() {
         return getCursor().getParentOrThrow().getValue() instanceof Yaml.Document ||
-               getCursor().getParentOrThrow(2).getValue() instanceof Yaml.Document;
+                getCursor().getParentOrThrow(2).getValue() instanceof Yaml.Document;
```

Comment:
I'm seeing your IDE introduce these extra spaces where we tend to have the continuation on the exact same line. Could you double check your settings?
```suggestion
               getCursor().getParentOrThrow(2).getValue() instanceof Yaml.Document;
```

Reply:
I have the default settings:

<img width="695" alt="image" src="https://github.com/user-attachments/assets/ec9ac74b-0c07-4cf2-9805-97bc0e2b9530">

If I change one thing, the "Default" label turns to blue (even after restarting the IDE itself). I did also "Restore Defaults" and it didn't change a thing. I even downloaded IntelliJ IDEA 2024.3 just now, still no difference. So I am pretty sure I use the default settings myself.

Are you sure about yours 😬?